### PR TITLE
fix(dialog): better id management for header title

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

-----
-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2641/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/14326/ui-diff)

File: packages/ng/dialog/dialog-header/dialog-header.component.ts [hunk 1]
```diff
@@ -43,16 +40,14 @@ export class DialogHeaderComponent implements OnInit {
 	ngOnInit(): void {
 		// Using setTimeout here to make sure this will be handled in the next Cd cycle, not the current one.
 		setTimeout(() => {
-			if (!this.id) {
-				this.id = `lu-dialog-header-${nextId++}`;
-			}
 			const header = this.#elementRef.nativeElement.querySelector('h1');
+			const id = header.id || `lu-dialog-header-${nextId++}`;
```
Comment (GuillaumeNury) [inline packages/ng/dialog/dialog-header/dialog-header.component.ts:44]:
```suggestion
			const id = header?.id || `lu-dialog-header-${nextId++}`;
```

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2641/storybook

**Review by shajz — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/14364/ui-diff)

**Pull request merged**